Theme six new branch

JankaRynd opened the pull request.

Commit:
dobavila zagotovky dla zadania

Commit:
perepisala polnostju vse!!

warlife3 commented on yana/homework/ts/HomeWorkTS.java:
```diff
@@ -0,0 +1,65 @@
+                System.out.println( ( a + 1 ) + ". " + answerOptions[i][a]);
+            }
+            System.out.print("Ваш ответ: ");
+            int result = scanner.nextInt();
```
Смотри если тут ввести что то кроме цифр то программа упадет. Пример: 1qwe
можешь это исправить?

warlife3 commented on yana/homework/ts/HomeWorkTS.java:
```diff
@@ -0,0 +1,65 @@
+        // Ниже вместо null надо написать реализацию,
+        // написано так, чтобы просто компилировалось
+
+        String[] questions = {
```
Так же могу накидать тебе еще пару заданий повышенной сложности. Если у тебя есть желание и время то напиши мне -)))

Commit:
Сделала дополнительное задание!!!!!

Commit:
Сделала дополнительное задание!!!!!
и исправила код что бы при вводе неправильного значение пользователь об этом узнал
